launcher/src/main/java/io/snappydata/tools/QuickLauncher.java

```diff
@@ -164,7 +165,16 @@ private int start(final String[] args) throws IOException, InterruptedException
     // See bug 32574.
     String msg = verifyAndClearStatus();
     if (msg != null) {
+      final Status status = getStatus();
```

Comment:
Better to avoid getting the status again here by having verifyAndClearStatus return the exit code and do the System.err.println() itself. Here a non-zero return code will be checked and returned. The same change of using the proper exit code for System.exit() call can be also done in CacheServerLauncher for cases where QuickLauncher is not being used (though will only happen with a special flag but handling in both places will be good for consistency).

Reply:
ok.